### ISSUE TITLE
Metashape version is not matching with the version of requirements.txt 

### ISSUE THREAD

**Issue by debpal** [s-du/IRMapper#10]
When I trying to install through pip, the Metashape module cannot be installed. The current Metashape version is 0.0.4 and the requirement file contains 2.0.1. I have attached the screenshot. 
![Capture](https://github.com/s-du/IRMapper/assets/48341250/fb5e1986-1b85-4655-8ce8-855e1edf2466)
What should I do?

**Comment by s-du** [comment]
Hi, I realize I should remove Metashape from the requirements. Normally it is installed automatically when the main.py is launched for the first time. A specific module installation file is included in the ‘resources’ files and is fetched at runtime. Does the app starts?

**Comment by debpal** [comment]
> Hi, I realize I should remove Metashape from the requirements. Normally it is installed automatically when the main.py is launched for the first time. A specific module installation file is included in the ‘resources’ files and is fetched at runtime. Does the app starts?

No, the app does not start. In the previous attachment, you can see that PySide6 is successfully installed. Now is is showing error 'No module named PySide6'. The proof is attached.
![Capture1](https://github.com/s-du/IRMapper/assets/48341250/5cce3e9d-cfe9-4d3f-a0b5-3a75cd9061c1)

**Comment by s-du** [comment]
It seems that you have installed everything in a virtual env (first screenshot) but you try to launch the app on the main env. You would need to activate the virtual env first. There is also a windows installer in the ‘releases’ section if you want to test the app more easily! Kind regards

**Comment by debpal** [comment]
> It seems that you have installed everything in a virtual env (first screenshot) but you try to launch the app on the main env. You would need to activate the virtual env first. There is also a windows installer in the ‘releases’ section if you want to test the app more easily! Kind regards
Thanks for pointing out the matter. I have activated the irmapper env, but gettting the same error. The screenshot is attached.
![Capture2](https://github.com/s-du/IRMapper/assets/48341250/eca79952-bb02-440c-8f23-de986e84d9a3)
